feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/domain/usecase/GetSpecialFolderOptions.kt

```diff
@@ -24,6 +24,10 @@ class GetSpecialFolderOptions(
                 ?: error("No incoming server settings available")
 
             val remoteFolders = folderFetcher.getFolders(serverSettings, authStateStorage)
+                .sortedWith(
+                    compareByDescending<RemoteFolder> { it.type == FolderType.INBOX }
+                        .thenBy(String.CASE_INSENSITIVE_ORDER) { it.displayName },
```

Comment:
This moves the INBOX to the third position in the list of folders, why is that necessary?

Reply:
It's done for consistency with the existing special folder preferences under *Settings → [Account] → Folders*. I believe we originally did this because that's also what we do in the folder list.

We can change the sort order later. Probably best to do it when we rewrite the settings screen. Then we can also share the logic to sort the list.